sceptre/diffing/diff_writer.py

```diff
@@ -207,3 +208,21 @@ def dump_diff(self, diff: List[str]) -> str:
         # Difflib doesn't care about the output format since it only outputs strings. We would have
         # accounted for the output format in the differ itself rather than here.
         return '\n'.join(diff)
+
+
+class ColouredDiffLibWriter(DiffLibWriter):
+    """A DiffWriter for StackDiffs where the DiffType is a a list of strings with coloured diffs."""
+
+    def _colour_diff(self, diff):
```

Comment:
Let's add the annotation just for consistency here. 
```suggestion
    def _colour_diff(self, diff: List[str]):
```